docs/contract.md

```diff
@@ -0,0 +1,45 @@
+# Our Contract
+
+We revised our contracts early in 2018 to include more specific language about what's expected from both sides,
+such as clearer milestones and how quickly we should provide feedback on updates to your content.
+This page is a simplified human-readable summary of its terms,
+and is not a substitute for the full agreement.
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
+(not least because it includes delivery dates).
+
+The contract's key features are:
+
+- You grant us a perpetual non-exclusive license to use the material that you develop for us.
+  The "non-exclusive" part means that you can use the material elsewhere,
+  e.g., in other courses or workshops.
```

Comment:
Do we still ask the instructor to notify us if they do that?

Reply:
Yes, but I thought that adding this would be getting into the weeds - happy to do so.